frontend/cypress/support/component.js

```diff
@@ -0,0 +1,27 @@
+// ***********************************************************
+// This example support/component.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import '../../../cypress/support/commands'
```

Comment:
Por qual motivo você está importando o arquivo de comandos customizados de testes _end-to-end_ no arquivo de testes de componentes?